user_tools/tests/spark_rapids_tools_e2e/resources/scripts/common.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+readonly E2E_TEST_TOOLS_TMP_DIR="/tmp/spark_rapids_tools"
```

Comment:
/tmp is sometimes small or can be removed so it might be nice to document this env variable in the readme on running the tests (assuming user can override when running the tests)